lib/core/src/server/dev-server.js

```diff
@@ -53,6 +53,7 @@ export default function(options) {
             error.error = err;
             error.close = true;
             error.stats = stats;
+            stats.toJson(options.errorDetails);
```

Comment:
this is the only change?

Could you please explain what exactly this does?

Reply:
it's enabling the option errorDetails through the function toJson.
So if the managerConfig has an error it will provide information about the error

Reply:
does `stats.toJson` have side effects?
or does it mutate the given `options.errorDetails`
?

Reply:
it mutates the given options.errorDetails